pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -1327,22 +1118,20 @@ def export_mesh_obj(self, setup_name=None, intrinsic_dict={}):
 
     @aedt_exception_handler
     def plot_model_obj(
```

Comment:
The Return on this method is incorrect.

Reply:
I was doing exactly the same :-).
I've also added the possibility to show or not the plot.